apps/remix-ide/src/app/panels/tab-proxy.js

```diff
@@ -169,6 +169,9 @@ export class TabProxy extends Plugin {
     this.on('manager', 'pluginDeactivated', (profile) => {
       this.removeTab(profile.name)
     })
+
+    this.themeQuality = (await this.call('theme', 'currentTheme') ).quality
```

Comment:
Shouldn't this be in a try/catch? Because if the plugin call fails, it may break the UI.